### ISSUE TITLE
A11y: Form Validation

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5259]
Currently, our storefront form validation has several issues, particularly when it comes to multiple input errors. When there are multiple fields that have errors, only the first is highlighted with a text based indicator. The rest only use a different color outline, which is a failure of SC 1.4.1.[https://www.w3.org/WAI/WCAG22/Understanding/use-of-color|https://www.w3.org/WAI/WCAG22/Understanding/use-of-color|smart-link] 

This is also problematic when using a screen reader. VoiceOver will only read the tooltip ‘Please fill in this field’ of the first error. You don’t have any information if there is more than one error, and you don’t even have any information which field has an error - you have to use context clues from the navigation.

{panel:bgColor=#deebff}
All storefront facing features will need to be updated to reflect this solution
{panel}

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39235**



[created from NEXT-39235, comment 506368]

**Comment by shopwareBot** [comment]
[public] 

 > Status Change: **Open**

 This issue is ready for implementation.

 

[created from NEXT-39235, comment 510206]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.0**

 This issue will be / was solved in Shopware version ***6.6.10.0***.



[created from NEXT-39235, comment 535054]
